### ISSUE TITLE
training code

### ISSUE THREAD

**Issue by andriyrizhiy** [jannctu/TIN#5]
Can you provide training code?

**Comment by xavysp** [comment]
And the loss please, it looks like the loss is similar to HED, isn't it?
